src/components/NavBarItem/NavBarItem.tsx

```diff
@@ -0,0 +1,9 @@
+interface NavBarItemProps {
+  label: string;
+  isActive: boolean;
+}
+const NavBarItem = ({ label, isActive = false }: NavBarItemProps) => {
```

Comment:
since `isActive` is required boolean prop, I think you no need to declare default value there